### DISCUSSION TITLE
Both LDAP authentication with TLS encryption and without

### DISCUSSION THREAD

**Discussion by dhia-gharsallaoui** [trinodb/trino#18065] [Q&A]
Hello mates :wave: 
I am deploying Trino with LDAP authentication with TLS encryption managed by the load balancer as recommended. In my use case I have two kinds of consumers first are the ones outside the cluster which they pass by the load balancer all work fine. Second, the inside cluster consumers connect to Trino using the local network interface. In this case, they don't pass by the load balancer and we don't need encryption which is not compatible with the current setup.
My question is it possible to have both a secure port that enforces TLS encryption and another one for local communication that doesn't need encryption also it will be possible to use LDAP in it?
Or if there is any way to work around this.
Thank you

**Comment by luhea** [comment]
Hello.I config ldap in load balancer（haproxy） not success，Can you shared your config? thank you
